jax/_src/interpreters/partial_eval.py

```diff
@@ -1857,6 +1857,10 @@ def process_primitive(self, primitive, tracers, params):
   def default_process_primitive(self, primitive, tracers, params):
     avals = [t.aval for t in tracers]
     out_avals, effects = primitive.abstract_eval(*avals, **params)
+    if not (isinstance(out_avals, (tuple,list)) == primitive.multiple_results):
+      raise ValueError(str(primitive) + ".abstract_eval() method should return"
```

Comment:
Please add a test

Reply:
I don't find the current JAX custom call tests.
Where should I add a test?

Reply:
I added a test.